public/app/services/settingsService.js

```diff
@@ -13,17 +13,22 @@ angular.module('kibibitCodeEditor')
     var isFullscreen = false;
     var currentUndoManager = undefined;
     var currentEditor = undefined;
+    var modelist = ace.require('ace/ext/modelist');
```

Comment:
Please add as global angular constants both `modelist` and `themelist`.

constants are usually written in uppercase like so:

```javascript
angular.module('kibibit', [])
.constant("EDITOR", {
        "MODE_LIST": ace.require('ace/ext/modelist'),
        "THEME_LIST": ace.require('ace/ext/themelist')
    })
```